.github/prep-hotfix-sources.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash -x
+
+HOTFIX_TAG=$1
+REPO_PATH=$2
+SPEC_PATH=$3
+
+# Create OSG build dirs
+build_dir=$(pwd)/_build_dir
+for dirname in osg upstream; do
+    mkdir -p $build_dir/$dirname
+done
+
+version=$(echo "$HOTFIX_TAG" | sed "s/v\([0-9]\+\.[0-9]\+\.[0-9]\+\).*/\1/")
+# Remove illegal dashes in release string
+release=$(tr '-' '.' <<< 1.${HOTFIX_TAG#*-})
+sed "s/__VERSION__/$version/" "$SPEC_PATH" | \
+    sed "s/__RELEASE__/$release/" > \
```

Comment:
I know this is wild, but, you can put multiple sed expressions in the same command:

```suggestion
sed -e "s/__VERSION__/$version/" \
    -e "s/__RELEASE__/$release/" "$SPEC_PATH" > \
```

Reply:
this is life changing information